internal/store/credential-helpers/pass.go

```diff
@@ -157,23 +132,23 @@ func listPassDir(recurse bool, includeDirs bool, relative bool, args ...string)
 	passDir := getPassDir()
 	dirPath := path.Join(append([]string{passDir}, args...)...)
 	paths = []string{}
-	if !recurse {
+	if !recurse { //nolint: nestif // TODO: refactor.
```

Comment:
does it work with a space? I had troubles with linter with a space after `nolint:`

Reply:
yes, it works, but you are probably right, it should be `//nolint:nestif // <comment>`, but that things needs to be refactored and probably will be refactored soon, so I guess we can keep it as it is for now :)